Dumper/Utils.h

```diff
@@ -547,6 +547,46 @@ inline T* FindAlignedValueInProcess(T Value, const std::string& Sectionname = ".
 	return Result;
 }
 
+template<bool bShouldRelove32BitJumps = true>
```

Comment:
I assume this is a typo and should be `bShouldResolve32BitJumps`

Reply:
it is